controller/state/statedb.go

```diff
@@ -298,68 +292,56 @@ func (s *stateDB) TaskHistory(ctx context.Context, taskID string) ([]tasks.TaskE
 }
 
 func (s *stateDB) createInitialTasks(ctx context.Context) error {
-	err := s.saveTask(ctx, &tasks.AuthenticatedTask{
-		Task: tasks.Task{
-			UUID:   uuid.New().String()[:8],
-			Status: tasks.Available,
-			RetrievalTask: &tasks.RetrievalTask{
-				Miner:      "t01000",
-				PayloadCID: "bafk2bzacedli6qxp43sf54feczjd26jgeyfxv4ucwylujd3xo5s6cohcqbg36",
-				CARExport:  false,
-			},
+	err := s.saveTask(ctx, &tasks.Task{
+		UUID:   uuid.New().String()[:8],
+		Status: tasks.Available,
+		RetrievalTask: &tasks.RetrievalTask{
+			Miner:      "t01000",
+			PayloadCID: "bafk2bzacedli6qxp43sf54feczjd26jgeyfxv4ucwylujd3xo5s6cohcqbg36",
+			CARExport:  false,
 		},
-		Signature: []byte{},
 	})
 	if err != nil {
 		return err
 	}
 
-	err = s.saveTask(ctx, &tasks.AuthenticatedTask{
-		Task: tasks.Task{
-			UUID:   uuid.New().String()[:8],
-			Status: tasks.Available,
-			RetrievalTask: &tasks.RetrievalTask{
-				Miner:      "t01000",
-				PayloadCID: "bafk2bzacecettil4umy443e4ferok7jbxiqqseef7soa3ntelflf3zkvvndbg",
-				CARExport:  false,
-			},
+	err = s.saveTask(ctx, &tasks.Task{
+		UUID:   uuid.New().String()[:8],
+		Status: tasks.Available,
+		RetrievalTask: &tasks.RetrievalTask{
```

Comment:
these default tasks have moved to json in https://github.com/filecoin-project/dealbot/pull/65